plugin_tests/client/mockVGL.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable camelcase */
+/* eslint-disable underscore/prefer-constant */
```

Comment:
@ronichoudhury I see, I have another PR to bump the version. I explicitly added this file because I encountered errors when writing the tests.